flightlog/aerolist.php

```diff
@@ -1,311 +1,311 @@
+	'EDTQ'=> array('PATTONVILLE',"l.rec='EDTQ' AND l.alt < 979 AND (l.lat BETWEEN 48.85881 AND 48.86964 ) AND (l.lon BETWEEN 9.20921 AND 9.24011 )",279,''),
+	'LFHD'=> array('PIERRELATTE',"l.rec='LFHD' AND l.alt < 759 AND (l.lat BETWEEN 44.38509 AND 44.40885 ) AND (l.lon BETWEEN 4.70249 AND 4.72768 )",59,''),		
+	'LFRP'=> array('PLOERMEL Loyat',"l.rec='LFRP' AND l.alt < 772 AND (l.lat BETWEEN 47.99956 AND 48.00447 ) AND (l.lon BETWEEN -2.38809 AND -2.37 )",72,''),
+	'LKPL'=> array('PLZEN-LETKOV',"l.rec='LKPL' AND l.alt < 1119 AND (l.lat BETWEEN 49.72163 AND 49.72504 ) AND (l.lon BETWEEN 13.44711 AND 13.46043 )",419,''),	
+	'EPKP'=> array('POBIEDNIK',"l.rec='EPKP' AND l.alt < 901 AND (l.lat BETWEEN 50.0829 AND 50.09477 ) AND (l.lon BETWEEN 20.19089 AND 20.21312 )",201,''),
+	'POCK'=> array('Pocklington',"l.rec='UKPOC' AND l.alt < 727 AND (l.lat BETWEEN 53.91052 AND 53.94206 ) AND (l.lon BETWEEN -0.8294 AND -0.77163 )",27,''),	
+	'EPOP'=> array('POLSKA NOWA WIES',"l.rec='EPOP' AND l.alt < 888 AND (l.lat BETWEEN 50.62521 AND 50.63955 ) AND (l.lon BETWEEN 17.76729 AND 17.793 )",188,''),
+	'LFSV'=> array('PONT SAINT VINCENT',"l.rec='LFSV' AND l.alt < 1107 AND (l.lat BETWEEN 48.59432 AND 48.60714 ) AND (l.lon BETWEEN 6.0482 AND 6.09133 )",407,''),	
 	'LFGO'=> array('PONT SUR YONNE',"l.rec='LFGO' AND l.alt < 771 AND (l.lat BETWEEN 48.28417 AND 48.29721 ) AND (l.lon BETWEEN 3.23484 AND 3.25284 )",71,''),
-	'PORT'=> array('Portmoak',"l.rec='Portmoak' AND l.alt < 810 AND (l.lat BETWEEN 56.184498 AND 56.197309 ) AND (l.lon BETWEEN -3.34453 AND -3.313957 )",110,''),
-	'LZPE'=> array('PRIEVIDZA',"l.rec='LZPE' AND l.alt < 960 AND (l.lat BETWEEN 48.75943 AND 48.77225 ) AND (l.lon BETWEEN 18.57778 AND 18.59604 )",260,''),
-	'LFTP'=> array('PUIMOISSON',"l.rec='LFTP' AND l.alt < 1471 AND (l.lat BETWEEN 43.86327 AND 43.87234 ) AND (l.lon BETWEEN 6.15561 AND 6.17295 )",771,''),	
-	'LFNW'=> array('PUIVERT',"l.rec='LFNW' AND l.alt < 1191 AND (l.lat BETWEEN 42.90532 AND 42.91614 ) AND (l.lon BETWEEN 2.04651 AND 2.06258 )",491,''),
-	'LKRA'=> array('RANA',"l.rec='LKRA' AND l.alt < 969 AND (l.lat BETWEEN 50.40246 AND 50.40946 ) AND (l.lon BETWEEN 13.73945 AND 13.76037 )",269,''),	
-	'LFQA'=> array('REIMS Prunay',"l.rec='LFQA' AND l.alt < 795 AND (l.lat BETWEEN 49.20463 AND 49.21261 ) AND (l.lon BETWEEN 4.14573 AND 4.16903 )",95,''),
-	'LOLK'=> array('RIED-KIRCHHEIM',"l.rec='LOLK' AND l.alt < 1120 AND (l.lat BETWEEN 48.2092 AND 48.21527 ) AND (l.lon BETWEEN 13.33668 AND 13.3529 )",420,''),	
+	'PORT'=> array('Portmoak',"l.rec='Portmoak' AND l.alt < 810 AND (l.lat BETWEEN 56.184498 AND 56.197309 ) AND (l.lon BETWEEN -3.34453 AND -3.313957 )",110,''),
+	'LZPE'=> array('PRIEVIDZA',"l.rec='LZPE' AND l.alt < 960 AND (l.lat BETWEEN 48.75943 AND 48.77225 ) AND (l.lon BETWEEN 18.57778 AND 18.59604 )",260,''),
+	'LFTP'=> array('PUIMOISSON',"l.rec='LFTP' A
```

Comment:
'EDPJ'=> array('LAICHINGEN',"l.rec='EDPJ' AND l.alt < 1500 AND (l.lat BETWEEN 48.49120 AND 48.50380 ) AND (l.lon BETWEEN 9.623980 AND 9.656570)",742,''),